RELEASING.md

```diff
@@ -32,3 +32,11 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 ## Releasing
 
 The release process is standard across repositories in this org and is run by a release manager volunteering from amongst [MAINTAINERS](MAINTAINERS.md).
+
+1. Create a tag, e.g. `v2.1.0`, and push it to the GitHub repo.
+2. The [opensearch-java-maven-sign-and-release/](https://build.ci.opensearch.org/job/opensearch-java-maven-sign-and-release/) will be automatically kicked off.
+3. Login to [AWS OSS Sonatype](https://aws.oss.sonatype.org/#stagingRepositories), and locate the staged release.
```

Comment:
@gaiksaya the @opensearch-project/engineering-effectiveness team uses an account for this with the right permissions - is the username public? can we put it here?

Reply:
I do not believe its public. So no